index.js

```diff
@@ -104,6 +105,19 @@ module.exports = function wss({registerErrors}) {
                     wss.emit('connection', ws, request);
                 });
             });
+
+            this.config.k8s = {
+                ports: [].concat(this.config.namespace).map((namespace) => ({
+                    name: 'http-' + namespace.replace(/\//, '-').toLowerCase(),
+                    service: true,
+                    ingress: this.config.rooms.map(room => ({
```

Comment:
Is there a real need for ingress for each room? 
They all pont to the same service...
I think one ingress at `/wss/${namespace}` is enough.